.github/workflows/build-ios.yml

```diff
@@ -24,6 +24,7 @@ jobs:
           rustup toolchain install nightly
           rustup +nightly target add aarch64-apple-ios
           rustup +nightly target add x86_64-apple-ios
+          rustup default nightly
```

Comment:
Was this needed for something? Doesn't matter too much, although it shouldn't be required.

Reply:
It is required for the rust cache action. See commit d930819d3bb2d975378408f131863abee4631896.